### PR TITLE
Don't crash if runtime is Google App Engine

### PR DESCRIPTION
**Pull request by ilanbiala**

### CODE REVIEW

File: src/main/java/com/twilio/http/NetworkHttpClient.java [hunk 1]
```diff
@@ -45,13 +45,22 @@ public NetworkHttpClient() {
             new BasicHeader(HttpHeaders.ACCEPT_ENCODING, "utf-8")
         );
 
-        client = HttpClientBuilder.create()
-            .useSystemProperties()
+        String googleAppEngineVersion = System.getProperty("com.google.appengine.runtime.version");
+        boolean isGoogleAppEngine = googleAppEngineVersion != null && !googleAppEngineVersion.isEmpty();
```
Comment (efossier) [inline src/main/java/com/twilio/http/NetworkHttpClient.java]: Replace with `Strings.isNullOrEmpty()`

**Comment by katafractari** [comment]
@efossier Tested on GAE **Java 7** platform. I can confirm it's working.

**Comment by efossier** [comment]
@katafractari thanks! we'll get this into the next release

**Comment by katafractari** [comment]
@efossier Cool, thanks! Do you maybe have a rough date for the release of the next version?

**Comment by efossier** [comment]
@katafractari We are targeting today but if not sometime this week.

**Pull request merged**